Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -19,10 +19,28 @@ import NIOHTTP1
 /// information.
 public struct HPACKHeaders {
     internal var headers: [HPACKHeader]
-    
+
+    // see 8.1.2.2. Connection-Specific Header Fields in RFC 7540
+    let illegalHeaders: [String] = ["connection", "keep-alive", "proxy-connection", "transfer-encoding", "upgrade"]
```

Comment:
I think strictly this is a `static`, though I don't know if we gain much by writing that down.

Reply:
done, thanks